easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-foss-2016a.eb

```diff
@@ -64,7 +67,8 @@ configopts += '-DWITH_IPP=ON '
 configopts += '-DENABLE_SSE=ON -DENABLE_SSE2=ON -DENABLE_SSE3=ON '
 configopts += '-DWITH_CUDA=OFF '
 
-postinstallcmds = ["cp 3rdparty/ippicv/unpack/ippicv_lnx/lib/intel64/libippicv.*  %(installdir)s/lib"]
+postinstallcmds = ["cp %(builddir)s/opencv-3.1.0/3rdparty/ippicv/unpack/ippicv_lnx/lib/intel64/libippicv.*  "
```

Comment:
@Flamefire using `opencv-%(version)s` to avoid hardcoding the version here (same applies to other easyconfigs)?

Reply:
Done: https://github.com/easybuilders/easybuild-easyconfigs/pull/9628/commits/79ac36111840cb00dd8d605860f7a0616e3840e4